vachaar-next/utils/request.ts

```
@@ -1,23 +1,23 @@
 import toast from "react-hot-toast";
 
-const isDev = process.env.NODE_ENV === "development";
-const baseUrl = isDev ? "http://localhost" : process.env.NEXT_PUBLIC_BASE_URL;
-
 export const makeRequest = (
   endpoint: string,
-  options: { method: string; body: Object },
+  options: { method: string; body?: Object; formData?: FormData },
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)